k2/csrc/ragged_ops.h

```diff
@@ -1361,6 +1361,14 @@ Ragged<int32_t> UniqueSequences(Ragged<int32_t> &src);
 template <typename T>
 void SegmentedExclusiveSum(Ragged<T> &src, Array1<T> *dst);
 
+/*
+  Construct a Ragged with 2 axes.
+    @param [in] vecs  Input of a two dimensional list
```

Comment:
I would add documentation like below to clarify,
 `vecs.size()` is the number of rows of the returned `ans`, i.e. `ans.Dim0() == vecs.sizes()`, and `vecs[i]` contains the elments for row i in `ans`.